qa/rpc-tests/wallet_mergetoaddress_2.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import shutil
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal, initialize_chain_clean, \
+    start_node, connect_nodes_bi, sync_blocks, sync_mempools, \
+    wait_and_assert_operationid_status
+
+import time
+from decimal import Decimal
+from distutils.dir_util import copy_tree
+
+class WalletMergeToAddressTest (BitcoinTestFramework):
+
+    op_result_k = "op_result"
+    balance_delta_k = "balance_delta"
+    joinsplit_len_k = "joinsplit_len"
+    mergingUTXOs_k = "mergingUTXOs"
+    mergingNotes_k = "mergingNotes"
+    mergingTransparentValue_k = "mergingTransparentValue"
+    mergingShieldedValue_k = "mergingShieldedValue"
+
+    def import_data_to_data_dir(self):
+        #importing datadir resource
+        import os
+        os.chdir(os.path.dirname(__file__))
+        resource_file = os.getcwd() + "/resources/wallet_mergetoaddress_2/test_setup_.zip"
+        import zipfile
+        with zipfile.ZipFile(resource_file, 'r') as zip_ref:
+            zip_ref.extractall(self.options.tmpdir)
+        copy_tree(self.options.tmpdir + "/test_setup_", self.options.tmpdir)
+        shutil.rmtree(self.options.tmpdir + "/test_setup_")
```

Comment:
`import os` and `import zipfile` can be moved as global import. Also, be removing the `test_setup_` as root of the zip file (i.e. the root of the folder will contain the directories `node0`, `node1` and `node2`) it is possible to use the extractall() method without the need of do `chdir()`, `copy_tree()` and `rmtree()`.
Finally, using the `join()` function with `os.sep()` as join element will result in a more portable code
```suggestion
    def import_data_to_data_dir(self):
        #importing datadir resource
        resource_file = os.sep.join([os.path.dirname(__file__), 'resources', 'wallet_mergetoaddress_2', 'test_setup_.zip'])
        with zipfile.ZipFile(resource_file, 'r') as zip_ref:
            zip_ref.extractall(self.options.tmpdir)
```